app/models/applicant_detail.rb

```diff
@@ -46,6 +46,8 @@ class ApplicantDetail < ApplicationRecord
   validates :gender, presence: true
   validates :birthdate, presence: true
   validates :shirt_size, presence: true
+  validates :demographic, presence: true
+
```

Comment:
extra line can be removed

Reply:
Three empty lines are removed here [6259e36](https://github.com/lsa-mis/mmss-mysql/pull/138/commits/6259e36d39ebd573cd0daaf5c8e103774fd24921)